src/Common/tests/System/Collections/ICollection.NonGeneric.Tests.cs

```diff
@@ -208,7 +208,7 @@ public void ICollection_NonGeneric_CopyTo_TwoDimensionArray_ThrowsException(int
             }
         }
 
-        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
+        [ConditionalTheory("System.PlatformDetection, CoreFx.Private.TestUtilities!" + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
```

Comment:
@weshaggard if I get a clean CI run, are you ok to merge this PR change and I'll continue looking at your buildtool idea? 
I want to have PlatformDetection code moved from now if possible.